### PR TITLE
Update the planner list error state

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-13908
affects: Parent
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/0j2pvbc) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F0j2pvbc)
teacher: [Link](https://rebrand.ly/di6rgf4) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fdi6rgf4)
parent: [Link](https://rebrand.ly/l4a8ub6) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fl4a8ub6)

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1023839&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent

[MBL-13908](https://instructure.atlassian.net/browse/MBL-13908)

<details><summary>:x: XCTest failed: CoreTests/UIImageViewExtensionsTests/testLoadSvg</summary>


    Assertion Failure: UIImageViewExtensionsTests.swift:86: Asynchronous wait failed: Exceeded timeout of 2 seconds, with unfulfilled expectations: "load callback runs".
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Views/ListBackgroundView.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import UIKit
+
+public class ListBackgroundView: UIView {
```
Comment (armstrongnate) [inline Core/Core/Views/ListBackgroundView.swift]: How/where is this used?

**Pull request merged**